bench/micro/Bench/Database/LSMTree/Internal/Integration.hs

```diff
@@ -202,6 +203,15 @@ newtype Page f k = Page { getContents :: f k }
   deriving stock (Show, Generic, Functor)
   deriving anyclass NFData
 
+-- | TODO: enable larger-than-page values
```

Comment:
What does this TODO refer to exactly?

Reply:
The environment generated for this benchmark doesn't generate larger-than-page values, hence this function mapping to `AppendSinglePage` only. If we wanted to benchmark with larger-than-page values too, then one approach would be to change the generator code to look more like https://github.com/input-output-hk/lsm-tree/pull/55/files#diff-5a1295f649f00eff25730818dec4c25cc3fafb4d20234f6097d488153ee7cdc8R179

Reply:
Ok, I think that's fine for now. Our normal benchmark should probably be all single pages, and then we can have a separate microbenchmark for multi-pages, just to check the multi-page case isn't terrible in comparison. I don't think we need mixtures of big and small.

Reply:
Added a TODO